adcc/test_modified_transition_moments.py

```diff
@@ -32,7 +32,7 @@
 from pytest import approx
 
 # The methods to test
-methods = ["adc2"]
+methods = ["adc2", "cvs-adc1", "cvs-adc2"]
```

Comment:
Unfortunately it's not qutie just that simple in this test, because you need to construct a cvs-ReferenceState as well. See https://github.com/adc-connect/adcc/blob/master/adcc/testdata/cache.py#L130 versus https://github.com/adc-connect/adcc/blob/master/adcc/testdata/cache.py#L122. See e.g. https://github.com/adc-connect/adcc/blob/805d2a40de30a1556fc08a88bc86c7bace9f590c/adcc/test_AdcMatrix.py#L42 how you can solve this.